tests/libwallet_api_tests/main.cpp

```diff
@@ -63,29 +65,28 @@ const char * WALLET_PASS = "password";
 const char * WALLET_PASS2 = "password22";
 const char * WALLET_LANG = "English";
 
-// change this according your environment
-
-const std::string WALLETS_ROOT_DIR = "/home/mbg033/dev/monero/testnet/";
-
-const std::string TESTNET_WALLET1_NAME = WALLETS_ROOT_DIR + "wallet_01.bin";
-const std::string TESTNET_WALLET2_NAME = WALLETS_ROOT_DIR + "wallet_02.bin";
-const std::string TESTNET_WALLET3_NAME = WALLETS_ROOT_DIR + "wallet_03.bin";
-const std::string TESTNET_WALLET4_NAME = WALLETS_ROOT_DIR + "wallet_04.bin";
-const std::string TESTNET_WALLET5_NAME = WALLETS_ROOT_DIR + "wallet_05.bin";
-const std::string TESTNET_WALLET6_NAME = WALLETS_ROOT_DIR + "wallet_06.bin";
+std::string WALLETS_ROOT_DIR = "/var/monero/testnet_pvt";
```

Comment:
No, test is not creating these files - wallets needs to be pre-seed with some money, as this takes some time, I keep it in persistent directory. Added brief documentation on testing environment, please see tests/libwallet_api_tests/scripts/README.md. Improvements and suggestions are always welcome :)